tools/tools-core/src/test/java/org/wso2/carbon/tools/dropins/DropinsDeployerToolTest.java

```diff
@@ -215,41 +214,38 @@ private static void createOtherProfiles() throws IOException {
     }
 
     private static void prepareCarbonHomeForDropinsTests() throws IOException {
-        String dropinsFolder = Constants.DROPINS;
-        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, dropinsFolder,
+        String libFolder = Constants.LIB;
```

Comment:
What would be the reason for assigning a constant to a local variable without using the constant itself?